roles/disaster_recovery/examples/generator.log

```diff
@@ -0,0 +1,2152 @@
+DEBUG:root:* Trying 192.168.1.248:8080...
```

Comment:
Not sure if we should have the log in the collection.
Is it a necessary example?